content/blog/posts/at-the-cross-roads-of-risk-factors-how-the-duncan-unit-became-a-hotbed-for-covid19.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "At the Cross Roads of Risk Factors: How the Duncan Unit Became a Hotbed
+  for COVID19"
+date: 2021-02-04
+author: Margarita Bronshteyn
+hero: ""
+---
+<!--StartFragment-->
+
+In June of 2020, the executive director of the Texas Justice Initiative, Eva Ruth Moravec, started noticing a disturbing trend in the custodial death data - an alarming number of [announcements about COVID-related deaths](https://www.tdcj.texas.gov/covid-19/index2.html) were coming from the same unit. In fact, of the 48 incarcerated individuals who died from COVID19 in Texas detention centers in June 2020, 14 were from the Duncan Unit. Meaning, that a unit that accounted for [0.3% of the state prison population](https://gritsforbreakfast.blogspot.com/search?q=covid+release) was suddenly reporting nearly 30% of the COVID19 deaths that month. Together with Margarita Bronshteyn, one of TJI’s data scientists, Eva Ruth set out to understand what was driving this mortality rate and to shed light on those who died behind bars without a life sentence.
```

Comment:
That was a direct line from the previous medium post. The link seems to suggest the numbers are about the whole TDCJ population. Which I think could make sense. Though this article suggests that it might be just the prisons:
https://www.texastribune.org/2020/12/01/texas-prisons-close-understaffing/#:~:text=%E2%80%9CThe%20population%20is%20122%2C000%20and,population%20for%20TDCJ%20since%201995.%E2%80%9D

Reply:
I read that "in the TDCJ system as a whole" to include state jails too. I think we're good assuming that it's total

Reply:
ok then I will accept these changes!

Reply:
Actually quick question - do you think we should change the link on the .3% from the current link to the texastribune one?

Reply:
> Actually quick question - do you think we should change the link on the .3% from the current link to the texastribune one?

sure!